.circleci/config.yml

```diff
@@ -6,6 +6,7 @@ aliases:
   - &lint |
       go fmt ./...
       go get -u golang.org/x/lint/golint
+      go install golang.org/x/lint/golint
```

Comment:
not adding this line crashes the job execution with `golint: command not found`

Reply:
This setup is very strange.
I suggest that we create a backlog issue to add `make lint` target that will be called from this file. That way we can test it in local env as well.

Reply:
👍 -> https://github.com/qlik-oss/enigma-go/issues/281